tolulope-od/banka-react
# #167299555 Create Authenticated Navigation Bars

#### What does this PR do?

Creates Navigation Header and Side Menu for authenticated users
Writes tests for newly created components
Renames test files

#### Description of Task to be completed?
Create Navigation bars for authenticated users

#### How should this be manually tested?
After cloning the repo, `cd` into it, install dependencies by running `npm run dev` and visit the signup page on `localhost:8080` and register, you should be redirected to an authenticated user's page with the authenticated navigated headers showing

#### Any background context you want to provide?
N/a

#### What are the relevant pivotal tracker stories?
[#167299555](https://www.pivotaltracker.com/story/show/167299555)

#### Screenshots (if appropriate)
